src/sentry/event_manager.py

```diff
@@ -2418,19 +2418,37 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects: Projec
 
 @metrics.wraps("performance.performance_issue.should_create_group", sample_rate=1.0)
 def should_create_group(client: Any, grouphash: str, type: GroupType) -> bool:
-    times_seen = client.incr(f"grouphash:{grouphash}")
-    metrics.incr(
-        "performance.performance_issue.grouphash_counted",
-        tags={"times_seen": times_seen, "group_type": GROUP_TYPE_TO_TEXT.get(type, "Unknown Type")},
-        sample_rate=1.0,
-    )
+    with sentry_sdk.start_span(op="event_manager.should_create_group") as span:
+        times_seen = client.incr(f"grouphash:{grouphash}")
+        metrics.incr(
+            "performance.performance_issue.grouphash_counted",
+            tags={
+                "times_seen": times_seen,
+                "group_type": GROUP_TYPE_TO_TEXT.get(type, "Unknown Type"),
```

Comment:
@gggritso 

This metric was already in place for `grouphash_counted` so I probably shouldn't update it here for the sake of consistency with previous data. And then that brings up the question of whether I should change it for the new metric I added (`performance.performance_issue.issue_will_be_created`), since it won't be consistent with this one 😔 

Thoughts?

Reply:
Ah gotcha. I don't think `performance.performance_issue.issue_will_be_created` needs to be consistent with the current one, it just needs to be useful and helpful. I'm in the process of adding metrics right now that use `type.name`, so it'll be consistent with those!

Reply:
Up to you 🙏🏻

Reply:
I'll update it to use `type.name` then!